qualification/report/generate_pdf.py

```diff
@@ -886,9 +913,11 @@ def main():
     result_list = list_from_json(args.input)
     if args.commit_id:
         print(test_image_commit(result_list))
-    doc = document_from_list(result_list, args.report_doc_id)
+    procedure_doc, report_doc = document_from_list(result_list, args.report_doc_id, "E1200-0000-004")
     if args.pdf.endswith(".pdf"):
         args.pdf = args.pdf[:-4]  # Strip .pdf suffix, because generate_pdf appends it
+    report_name = args.pdf + "_report"
+    procedure_name = args.pdf + "_procedure"
```

Comment:
I'm not really happy with this. If you stick with having this script generate both reports in one go, have two command-line arguments so that the user gets to pick explicitly what to call both files.